src/khepri_cluster.erl

```diff
@@ -1235,6 +1236,21 @@ do_query_members(StoreId, RaServer, QueryType, Timeout) ->
                        [StoreId]),
                     Error
             end;
+        {error, Reason}
+          when ?HAS_TIME_LEFT(Timeout) andalso
+               (Reason == noconnection orelse
+                Reason == nodedown orelse
+                Reason == shutdown) ->
+            NewTimeout0 = khepri_utils:end_timeout_window(Timeout, T0),
+            NewTimeout = khepri_utils:sleep(
+                           ?NOPROC_RETRY_INTERVAL, NewTimeout0),
```

Comment:
Not totally related to this change but should we rename this constant to something like `TRANSIENT_ERROR_RETRY_INTERVAL` now that we use it for other error reasons?

Reply:
You are right. I can prepare another pull request.